### ISSUE TITLE
How many epochs does the vehicle dataset need to be trained?

### ISSUE THREAD

**Issue by Mike001-wq** [lutao2021/BrightDreamer#8]
Do all 600,000 training epochs in vehicle.sh need to be completed? I conduct experiment on 4 A40 and 800+hours are needed, much more than twice the 36 hours.

**Comment by lutao2021** [comment]
Sorry for this misunderstanding.

Actually, we didn't train the model 600,000 iterations. After you find there is no quality improvement, you could stop it manually.

**Comment by Mike001-wq** [comment]
> Sorry for this misunderstanding.
> 
> Actually, we didn't train the model 600,000 iterations. After you find there is no quality improvement, you could stop it manually.

Thank you very much for your reply. How many epochs can we stop after? Roughly speaking?

**Comment by lutao2021** [comment]
> > Sorry for this misunderstanding.
> > Actually, we didn't train the model 600,000 iterations. After you find there is no quality improvement, you could stop it manually.
> 
> Thank you very much for your reply. How many epochs can we stop after? Roughly speaking?

Maybe 200 epoches for IF, and another 100 epoches for IF+IF2.

It depends on your batch_size and learning rate.
